src/osp/OneStepProverHostIo.sol

```diff
@@ -132,7 +183,7 @@ contract OneStepProverHostIo is IOneStepProver {
         19103219067921713944291392827692070036145651957329286315305642004821462161904;
 
     // see: https://github.com/Layr-Labs/eigenda/blob/master/disperser/apiserver/server.go#L35
-    uint256 internal constant eigenDAMaxFieldElementsPerBlob = 268435456;
+    uint256 internal constant eigenDAMaxFieldElementsPerBlob = (2 * 1024 * 1024) / 32;
```

Comment:
why we need this variable?

Reply:
looks like it's no longer used. Removing.